Cannot find module '#pwa' or its corresponding type declarations.

I'm following the basic instructions to setup `@vite-pwa/nuxt`

`pnpm add @vite-pwa/nuxt -D
`

```
// nuxt.config.ts
import { defineNuxtConfig } from 'nuxt/config'

export default defineNuxtConfig({
  modules: [
    '@vite-pwa/nuxt'
  ],
  pwa: {
    /* PWA options */
  }
})
```

And got this error.

![image](https://github.com/vite-pwa/nuxt/assets/9573391/672f263c-6490-491d-885e-3fbaaad05c7d)
![image](https://github.com/vite-pwa/nuxt/assets/9573391/819dee49-36dd-4937-b1b0-1127ba35ef1f)


I also have tryed this and got the same error.
```
rm -Rf node_modules
rm -Rf dist
rm -Rf .nuxt
pnpm store prune
pnpm add @vite-pwa/nuxt -D --force  
```

Bumped into similar issue:
```
 ERROR
 ERROR(vue-tsc)  Cannot find module '#build/pwa-icons/pwa-icons' or its corresponding type declarations.
 FILE  ......./.nuxt/pwa-icons-plugin.ts:3:31

    1 | // Generated by @vite-pwa/nuxt
    2 | import { defineNuxtPlugin } from '#imports'
  > 3 | import type { PWAIcons } from '#build/pwa-icons/pwa-icons'
      |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    4 |
    5 | export default defineNuxtPlugin(() => {
    6 |   return {
```
After some `@nuxt/kit` version update (`3.14.x`? brought by something else) `#build` alias was not inserted in `tsconfig.json` in `.nuxt` anymore.

Downgrading `@nuxt/kit` down to `3.11.2` (matching current nuxt version) solved the issue for me.

--- a/src/runtime/composables/index.ts
+++ b/src/runtime/composables/index.ts
@@ -1,5 +1,6 @@
 import { computed, toValue } from 'vue'
-import type { MaybeRef } from 'vue'
+import type { MaybeRef, UnwrapNestedRefs } from 'vue'
+import type { PwaInjection } from '../plugins/types'
 import { useNuxtApp } from '#imports'
 import type { PwaTransparentImageProps } from '#build/pwa-icons/PwaTransparentImage'
 import type { PwaMaskableImageProps } from '#build/pwa-icons/PwaMaskableImage'
@@ -64,6 +65,9 @@ export function useApplePwaIcon(image: AppleImageType) {
 }
 export function useAppleSplashScreenPwaIcon(image: AppleSplashScreenImageType) {
   return usePWAIcon('appleSplashScreen', image)
+}
+export function usePWA(): UnwrapNestedRefs<PwaInjection> | undefined {
+  return useNuxtApp().$pwa
 }
 
 function usePWAIcon(

